Application/EdFi.Ods.WebApi/log4net.config

```diff
@@ -37,4 +37,18 @@
     <appender-ref ref="TraceAppender" />
     <appender-ref ref="AppInsightsAppender" />
   </root>
+  <logger name="RequestResponseDetailsLogger" additivity="false">
```

Comment:
both logger RequestResponseDetailsLogger and appender RequestResponseDetailsFileAppender must be configured for EdFi.Ods.Api.IntegrationTestHarness project is used to run the all postman intregations tests https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/main/Postman%20Test%20Suite. Please test all request types as well.